app/worker.php

```diff
@@ -216,17 +211,31 @@
 $platform = new Appwrite();
 $args = $_SERVER['argv'];
 
-if (isset($args[0])) {
-    $workerName = end($args);
-} else {
+if (!isset($args[1])) {
     Console::error('Missing worker name');
+    Console::exit(1);
+}
+
+\array_shift($args);
+$workerName = $args[0];
+$workerIndex = $args[1] ?? '';
+
+if (!empty($workerNum)) {
+    $workerName .= '_' . $workerIndex;
 }
 
 try {
+    /**
+     * Any worker can be configured with the following env vars:
+     * - _APP_WORKERS_NUM           The total number of worker processes
+     * - _APP_WORKER_PER_CORE       The number of worker processes per core (ignored if _APP_WORKERS_NUM is set)
+     * - _APP_CONNECTIONS_DB_QUEUE  The name of the queue to read for database events
```

Comment:
```suggestion
     * - _APP_QUEUE_NAME  The name of the queue to read for database events
```

It would be better to not have special conditions just for the database worker. If empty we can try and generate the queue name as it is today or not change it and overwrite it only when a specific value has been given.